go.mod

```diff
@@ -10,7 +10,7 @@ go 1.16
 
 require (
 	bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
-	github.com/Azure/azure-sdk-for-go v53.1.0+incompatible
+	github.com/Azure/azure-sdk-for-go v54.1.0+incompatible
```

Comment:
/close

in favor of https://github.com/kubernetes/kubernetes/pull/102441 which updates to v55